[2024-05-08T04:36:08.283+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-08T04:36:08.317+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_table_dump.import_task manual__2024-05-08T04:36:06.141095+00:00 [queued]>
[2024-05-08T04:36:08.327+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_table_dump.import_task manual__2024-05-08T04:36:06.141095+00:00 [queued]>
[2024-05-08T04:36:08.328+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-08T04:36:08.344+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_task> on 2024-05-08 04:36:06.141095+00:00
[2024-05-08T04:36:08.351+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_table_dump', 'import_task', 'manual__2024-05-08T04:36:06.141095+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dump_load_dag.py', '--cfg-path', '/tmp/tmp9w4kkxlu']
[2024-05-08T04:36:08.354+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask import_task
[2024-05-08T04:36:08.354+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1015) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-08T04:36:08.355+0000] {standard_task_runner.py:63} INFO - Started process 1016 to run task
[2024-05-08T04:36:08.398+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_table_dump.import_task manual__2024-05-08T04:36:06.141095+00:00 [running]> on host 60e0adda864f
[2024-05-08T04:36:08.484+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_table_dump' AIRFLOW_CTX_TASK_ID='import_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T04:36:06.141095+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-08T04:36:06.141095+00:00'
[2024-05-08T04:36:08.486+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-08T04:36:08.500+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-08T04:36:08.502+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mysql -u root -p1234 -h host.docker.internal -P 3306 fc_facts < /opt/***/dags/dump_file.sql']
[2024-05-08T04:36:08.509+0000] {subprocess.py:86} INFO - Output:
[2024-05-08T04:36:08.655+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-08T04:36:08.656+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-08T04:36:08.676+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=mysql_table_dump, task_id=import_task, execution_date=20240508T043606, start_date=20240508T043608, end_date=20240508T043608
[2024-05-08T04:36:08.731+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-08T04:36:08.750+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-08T04:36:08.753+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
